### PR TITLE
Various code improvements

### PR DESCRIPTION
**Pull request by MrDoomBringer**
mostly 1 and 0 to TRUE and FALSE

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
I lol'd

**Comment by ExcessiveUseOfCobblestone** [comment]
I lol'd

**Comment by MrDoomBringer** [comment]
@ExcessiveUseOfCobblestone shush its not done yet

File: code/game/machinery/doors/windowdoor.dm [hunk 1]
```diff
@@ -129,33 +129,33 @@
 	else
 		return 1
 
-/obj/machinery/door/window/open(forced=0)
-	if (src.operating == 1) //doors can still open when emag-disabled
+/obj/machinery/door/window/open(forced=FALSE)
+	if (operating == TRUE) //doors can still open when emag-disabled
```
Comment (Cyberboss) [inline code/game/machinery/doors/windowdoor.dm]: just if(operating)

File: code/game/machinery/doors/windowdoor.dm [hunk 2]
```diff
@@ -129,33 +129,33 @@
...
 	if(forced < 2)
 		if(obj_flags & EMAGGED)
 			return 0
-	if(!src.operating) //in case of emag
+	if(!operating) //in case of emag
 		operating = TRUE
 	do_animate("opening")
-	playsound(src.loc, 'sound/machines/windowdoor.ogg', 100, 1)
-	src.icon_state ="[src.base_state]open"
+	playsound(loc, 'sound/machines/windowdoor.ogg', 100, 1)
```
Comment (Cyberboss) [inline code/game/machinery/doors/windowdoor.dm]: loc => src

File: code/game/machinery/doors/windowdoor.dm [hunk 3]
```diff
@@ -129,33 +129,33 @@
...
+	icon_state ="[base_state]open"
 	sleep(10)
 
 	density = FALSE
-//	src.sd_set_opacity(0)	//TODO: why is this here? Opaque windoors? ~Carn
+//	sd_set_opacity(0)	//TODO: why is this here? Opaque windoors? ~Carn
```
Comment (Cyberboss) [inline code/game/machinery/doors/windowdoor.dm]: just delete this

File: code/game/machinery/doors/windowdoor.dm [hunk 4]
```diff
@@ -165,8 +165,8 @@
 			return 0
 	operating = TRUE
 	do_animate("closing")
-	playsound(src.loc, 'sound/machines/windowdoor.ogg', 100, 1)
-	src.icon_state = src.base_state
+	playsound(loc, 'sound/machines/windowdoor.ogg', 100, 1)
```
Comment (Cyberboss) [inline code/game/machinery/doors/windowdoor.dm]: loc => src

File: code/game/machinery/doors/windowdoor.dm [hunk 5]
```diff
@@ -181,7 +181,7 @@
 		if(BRUTE)
 			playsound(loc, 'sound/effects/glasshit.ogg', 90, 1)
 		if(BURN)
-			playsound(src.loc, 'sound/items/welder.ogg', 100, 1)
+			playsound(loc, 'sound/items/welder.ogg', 100, 1)
```
Comment (Cyberboss) [inline code/game/machinery/doors/windowdoor.dm]: loc => src

File: code/game/machinery/doors/windowdoor.dm [hunk 6]
```diff
@@ -266,12 +266,12 @@
 
 						var/obj/item/electronics/airlock/ae
 						if(!electronics)
-							ae = new/obj/item/electronics/airlock( src.loc )
+							ae = new/obj/item/electronics/airlock( loc )
```
Comment (Cyberboss) [inline code/game/machinery/doors/windowdoor.dm]: loc => drop_location()

**Review by Cyberboss — DISMISSED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**